packages/tree-view/src/ux-tree-view/ux-tree-view.html

```diff
@@ -0,0 +1,22 @@
+<template class="ux-tree-view">
+  <require from="./ux-tree-view.css"></require>
+  <ul class="ux-tree-view--list">
+    <li repeat.for="$node of nodes" class="ux-tree-view--list-item">
+      <let $tree.bind="element"></let>
+      <let $tree-vm.bind="$parent"></let>
+      <div class="ux-tree-view--node ${$node.selected ? 'ux-tree-view--node-selected' : ''}" click.delegate="nodeClicked($node)">
+        <ux-button if.bind="$node.children" type="fab" size="small" class="ux-tree-view--expander"
+          click.delegate="toggleExpanded($node, $event)">
```

Comment:
let's use `.trigger`, `.delegate` should only be used in app, as it could be interfered by shadowdom. Though we fixed it, it still feels better encapsulated with `.trigger`

Reply:
done